src/applications/disability-benefits/526EZ/components/ErrorableCheckboxes.jsx

```diff
@@ -0,0 +1,189 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'lodash';
+import classNames from 'classnames';
+import ExpandingGroup from '@department-of-veterans-affairs/formation/ExpandingGroup';
+
+// TODO: Move this component to the design system where these helpers have been moved
+function makeField(value, optionalDirty) {
+  const dirty = optionalDirty === undefined ? false : optionalDirty;
+  return { value, dirty };
+}
+
+/**
+ * A checkbox button group with a label.
+ *
+ * Validation has the following props.
+
+ * `additionalFieldsetClass` - String for any additional fieldset classes.
+ * `additionalLegendClass` - String for any additional legend classes.
+ * `label` - String for the group field label.
+ * `name` - String for the name attribute.
+ * `tabIndex` - Number for keyboard tab order.
+ * `options` - Array of options to populate group.
+ * `required` - is this field required.
+ * `values` - object. Values of the checkbox field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableCheckboxes extends React.Component {
```

Comment:
Can we reuse the errorable checkboxes in the design system? Instead of re-creating them here?

Reply:
This PR sets them up, they don't currently exist. There is another PR to move them into the design system once this is reviewed and merged.